connector/ldap/ldap.go

```diff
@@ -177,6 +178,7 @@ func (c *Config) openConnector(logger logrus.FieldLogger) (*ldapConnector, error
 		if field.val == "" {
 			return nil, fmt.Errorf("ldap: missing required field %q", field.name)
 		}
+		logger.Infof("ldap %s: %s", field.name, field.val)
```

Comment:
We might want to log all this stuff as info in case the user has mis-configured any field (even if the field isn't empty). Relates to https://github.com/coreos/dex/pull/1014#discussion_r131438929. Additionally I'm going to use the `missing` [array pattern](https://github.com/estroz/dex/blob/5749b86fc94294170bdb9abf48f88d8e944f2ff8/connector/saml/saml.go#L144) so we don't mask missing values with those preceding.